### PR TITLE
[IMP] Finnish translation

### PR DESCRIPTION
**Pull request by ossimantylahti**
*Description of the issue/feature this PR addresses:

Added and improved several Finnish translations for several modules.

*Current behavior before PR:

Several entries were missing Finnish language translations. Several entries had incorrect translations.

* Desired behavior after PR is merged:

Better Finnish localization

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Review by rdeodoo — CHANGES_REQUESTED** [review]
Hello and thank you for your contribution, it's really appreciated!

But it's not how to contribute to translations. You should do that in https://www.transifex.com/

For the explanation part, which might not be 100% accurate but you will get the big picture:
- We generate our translation terms (from our python/xml/js file eg) to `.pot` file **once** only when we freeze/release a version. (It's why we later have to not forget to manually modify this file if we need to change a wording in stable, because it won't be regenerated automatically)
- Someone (probably semi-automatically) is then exporting those `.pot` files to Transifex for the community to translate it on a regular basis (probably once a week?).
- Every week (Monday morning I think), someone is exporting those Transifex translations (which will have both new translation or translation fix) back to Odoo through/in `.po` file.

Here, it means that even if we were to merge what you have done, it would be most likely wiped away once we do the Transifex -> `.po` sync later.

tl;dr: You should contribute to translation on Transifex

Edit: see https://github.com/odoo/odoo/wiki/Contributing#how-can-i-translate-odoo-in-my-language

**Comment by rdeodoo** [comment]
The `semi automatically` "someone" is probably a mix between scripts / @C3POdoo / @mart-e 
He will be able to correct me if I wrongly explained part of our process :+1:

**Comment by mart-e** [comment]
No that's a good picture :)
In short, translations are made on Transifex
